include/fmt/compile.h

```diff
@@ -13,6 +13,12 @@
 
 #include "format.h"
 
+#if !defined(FMT_USE_NONTYPE_TEMPLATE_PARAMETERS) &&   \
+    defined(__cpp_nontype_template_parameter_class) && \
+    (!FMT_GCC_VERSION || FMT_GCC_VERSION >= 903)
+#  define FMT_USE_NONTYPE_TEMPLATE_PARAMETERS
+#endif
```

Comment:
Please make `FMT_USE_NONTYPE_TEMPLATE_PARAMETERS` defined to 1/0 for consistency with other similar macros and to make it possible to disable the feature. See e.g. https://github.com/fmtlib/fmt/blob/33f9a6d3609c7001afc7019c8308f4af8942ce0a/include/fmt/core.h#L178-L185

Reply:
Done